src/cromshell/status/command.py

```diff
@@ -57,23 +56,14 @@ def main(config, workflow_id):
         # TODO : Use this as a template for the Metadata subcommand
         # Get execution status count and filter the metadata down:
         request_meta_out = requests.get(
-            f"{config.cromwell_api_workflow_id}/metadata?{config.slim_metadata_parameters} "
+            f"{config.cromwell_api_workflow_id}/metadata?{config.slim_metadata_parameters}"
         )
 
         # tmp_metadata holds the workflow metadata as a dictionary
         tmp_metadata = json.loads(request_meta_out.content.decode("utf-8"))
-        execution_status_count = get_metadata_status_summary(tmp_metadata)
-
-        # Check for failure states:
-        failed = False
-        for task_status in execution_status_count:
-            for key in task_status:
-                if "Failed" in str(task_status[key]):
-                    failed = True
-                    break
 
         # Check for failures:
-        if not failed:
+        if not check_for_failure(tmp_metadata):
```

Comment:
I would rename this to be more of a statement ` if not workflow_failed` is clearer than `if not check_for_failure`